src/components/ProjectSearch/TextSearch.js

```diff
@@ -153,28 +160,23 @@ export default class TextSearch extends Component {
       }
     });
 
-    return (
-      <span className="line-value">
-        {matches}
-      </span>
-    );
+    return dom.span({ className: "line-value" }, ...matches);
```

Comment:
Ah, interesting, so using spreads for children is off the table for now I guess.

Thanks for the links